eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -164,46 +164,62 @@ function DeployStressPackage(
         if ($LASTEXITCODE) { return }
     }
 
-    $imageTag = "${registryName}.azurecr.io"
+    $imageTagBase = "${registryName}.azurecr.io"
     if ($repositoryBase) {
-        $imageTag += "/$repositoryBase"
+        $imageTagBase += "/$repositoryBase"
     }
-    $imageTag += "/$($pkg.Namespace)/$($pkg.ReleaseName):${deployId}"
-
-    $dockerFilePath = if ($pkg.Dockerfile) {
-        Join-Path $pkg.Directory $pkg.Dockerfile
+    $imageTagBase += "/$($pkg.Namespace)/$($pkg.ReleaseName):${deployId}"
+
+    $dockerFilePaths =@()
+    
+    $genValFile = Join-Path $pkg.Dockerfile "generatedValues.yaml"
```

Comment:
Doesn't this evaluate out to something like `foo/bar/stress/Dockerfile/generatedValues.yaml`?